bndtools/workspace
# Update Workspace template from OSGi R7 to R8

As suggested in https://github.com/bndtools/bndtools.workspace.min/pull/11#issuecomment-1832442153  by @scottslewis  we up this repo to OSGi R8 and base it on the files of bndtools.workspace.min initially

This may not be the final solution but at least a first step to have an OSGi R8 environment, so that templates may build upon that.

## How  to test this?

This is the place where I need support to verify if we are satisfied with result, or what needs to be changed / added.

- Go to Eclipse Preferences / BndTools / Workspace Template
- Remove the Github repositories (just for testing)
<img width="969" alt="image" src="https://github.com/bndtools/workspace/assets/188422/4a58cc74-fdf8-4a4a-a8d5-c1938ac1cc03">

- Add this current repo  to "Raw Git Clone URLs" (`https://github.com/chrisrueger/workspace.git` and branch `origin/R7-to-R8`)

- Click Apply and Close
- Go to File / New / Bnd OSGi Workspace
<img width="564" alt="image" src="https://github.com/bndtools/bndtools.workspace.min/assets/188422/39878671-3f12-4098-bad7-8d77ce64e691">

- Choose new Workspace or update the current one and Click next
<img width="1184" alt="image" src="https://github.com/bndtools/bndtools.workspace.min/assets/188422/48156f0b-80bd-4372-9afe-c9d43a580d46">

- Select the template from the repo you just entered above

<img width="1082" alt="image" src="https://github.com/bndtools/workspace/assets/188422/7cbb8bc2-10fc-4e5d-8670-d0a16f663b42">

- when asked to Switch to the Bnd Perpective say "Yes" or manually change there.
